bpf/generictracer/k_tracer.c

```diff
@@ -56,6 +56,32 @@ int BPF_KRETPROBE(beyla_kretprobe_sock_alloc, struct socket *sock) {
     return 0;
 }
 
+// Used by accept to grab the sock details
+SEC("kprobe/lock_sock_nested")
```

Comment:
We likely don't need the sock_alloc probe anymore, we should remove it in the bpf code and in the userspace.